### ISSUE TITLE
Native move up behavior implementation for FloatingActionsMenu

### ISSUE THREAD

**Issue by deep7176** [Clans/FloatingActionButton#375]
The floating actions menu doesn't move up with the snackbar when shown. This post has a way to work around that. It would be better if there was native functionality in this already out of the box. 

http://stackoverflow.com/questions/33217241/how-to-move-a-view-above-snackbar-just-like-floatingbutton

**Comment by prudhvirajkumar10** [comment]
Yes, the default action of the fab doesn't move up with the snackbar when shown
